sql_scripts/12_beers_f_filter.sql

```diff
@@ -0,0 +1,4 @@
+INSERT INTO beer.beers_filtered (id)
```

Comment:
Look into 11_beers_q_filter.sql and make the same adjustments here based on the comment left there